app/connectors/NpsConnector.scala

```diff
@@ -46,6 +47,7 @@ class NpsConnector @Inject()(http: HttpClient, applicationConfig: ApplicationCon
     http.GET[HttpResponse](url("/individuals/annual-tax-summary/" + ninoWithoutSuffix + "/" + TAX_YEAR)) recover {
       case _: BadRequestException => HttpResponse(BAD_REQUEST)
       case _: NotFoundException   => HttpResponse(NOT_FOUND)
+      case _: Upstream5xxResponse => HttpResponse(INTERNAL_SERVER_ERROR)
```

Comment:
Upstream5xxResponse and HttpResponse are deprecated

Reply:
done